pkg/operators/register.go

```diff
@@ -169,14 +445,65 @@ func setupFromConfig(cfg *setting.Cfg) (controllerCfg *controllerConfig, err err
 		TLSClientConfig: tlsConfig,
 	}
 
-	provisioningClient, err := client.NewForConfig(config)
+	client, err := client.NewForConfig(config)
```

Comment:
In the previous round, we defined `provisioningServerURL`. That won't scale too way now that we need folders, and dashboard APIs as well. I don't know if those need from something else either.